rllib/evaluation/worker_set.py

```diff
@@ -448,6 +448,7 @@ def sync_weights(
         to_worker_indices: Optional[List[int]] = None,
         global_vars: Optional[Dict[str, TensorType]] = None,
         timeout_seconds: Optional[int] = 0,
+        inference_only: Optional[bool] = False,
```

Comment:
Should we set this to True by default? Or at least add a TODO to switch this?

In the end - on the new API stack - all EnvRunners should only use the `inference_only` version. Also, there might be a time, in which we won't have a local worker anymore and simply use the Learners as a master-copy for all trainable weights.

Reply:
Good idea. Afaics any synching does only effect from learner to env-runner.